configsuite/schema.py

```diff
@@ -138,22 +139,65 @@ def _check_required_not_default(schema_level):
 }
 
 
-def assert_valid_schema(schema, allow_default=False, validate_named_keys=True):
-    _assert_valid_schema(schema, allow_default, validate_named_keys)
+def _build_meta_schema(deduce_required):
```

Comment:
If we're deducing required, the implemented validators for required will no longer apply...